packages/salesforcedx-vscode-core/src/commands/auth/orgLogout.ts

```diff
@@ -96,9 +96,9 @@ export async function orgLogoutDefault() {
     notificationService.showErrorMessage('Logout failed to run');
   } else if (username) {
     // confirm logout for scratch orgs due to special considerations:
-    // https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_logout.htm
-    const logoutCommandlet = new SfdxCommandlet(
-      new SfdxWorkspaceChecker(),
+    // https://developer.salesforce.com/docs/atlas.en-us.sf_dev.meta/sf_dev/sf_dev_auth_logout.htm
```

Comment:
Revert this URL.  It points to a blank page, and the old one URL now points to a page with the SF-style logout commands.